### ISSUE TITLE
floating legs?

### ISSUE THREAD

**Issue by nathanlycan** [BLOODWOLF333/Brutal-Doom-Community-Expansion#235]
### Version
~~v 21 gold~~ v21.5.0 (I assume?)

### Environment
computer

### Port
gzdoom

### Description
when i destroy the top of the hanging leg decoration it jumps down about a foot and just floats there
<img width="960" alt="Screenshot 2020-11-01 164556" src="https://user-images.githubusercontent.com/73799396/97816340-99f0cd00-1c62-11eb-80e0-d61edcb87c62.png">
![Uploading Screenshot 2020-11-01 164720.png…]()

**Comment by BLOODWOLF333** [comment]
![Screenshot_Doom_20201102_160141](https://user-images.githubusercontent.com/16950832/97924140-c9680e00-1d24-11eb-8b1a-50a275405602.png)
![Screenshot_Doom_20201102_160150](https://user-images.githubusercontent.com/16950832/97924145-cd942b80-1d24-11eb-82bb-c5c1b17d572a.png)

I fixed this last night and verified that it's working.  What is your load order?  How are you "pulling-in" the changes?

**Comment by nathanlycan** [comment]
I went into the pk3 file and typed the changed values into the dead.txt and when that didn't work i copy pasted the entire changed file on your website over the original as for load order i dont know but the only other wad besides doom i'm using is the music pack 5. It'll be a few days before I can get back to my computer and I'll mess with the load order then.Sent from my T-Mobile 4G LTE Device
-------- Original message --------From: Carr <notifications@github.com> Date: 11/2/20  5:06 PM  (GMT-05:00) To: BLOODWOLF333/Brutal-Doom-Community-Expansion <Brutal-Doom-Community-Expansion@noreply.github.com> Cc: nathanlycan <redmon9982@gmail.com>, Author <author@noreply.github.com> Subject: Re: [BLOODWOLF333/Brutal-Doom-Community-Expansion] floating legs?
  (#235) 


I fixed this last night and verified that it's working.  What is your load order?  How are you "pulling-in" the changes?

**Comment by nathanlycan** [comment]
If it helps i remember what I did to make the wads always load. i went into the gz doom config (the one with my computers name on it and on the always load part (i think it was called global load i had it written path= brutaldoom 21.5.0 then on the next line it had path= musicpack5. Or something along Those lines.  I think i reversed the order and had the same problem though.Sent from my T-Mobile 4G LTE Device
-------- Original message --------From: Carr <notifications@github.com> Date: 11/2/20  5:06 PM  (GMT-05:00) To: BLOODWOLF333/Brutal-Doom-Community-Expansion <Brutal-Doom-Community-Expansion@noreply.github.com> Cc: nathanlycan <redmon9982@gmail.com>, Author <author@noreply.github.com> Subject: Re: [BLOODWOLF333/Brutal-Doom-Community-Expansion] floating legs?
  (#235) 


I fixed this last night and verified that it's working.  What is your load order?  How are you "pulling-in" the changes?
